### PR TITLE
Bugfix/biometrickycagentmode

### PR DESCRIPTION
**Pull request by JNdhlovu**
## Summary

Agent mode was not set in biometric kyc view

## Known Issues

N/A

## Test Instructions
- Enable/Disable allow agent mode in  biometric kyc and it should show accordingly

## Screenshot
IMG_4846

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,11 @@
 ### Changed
 
 ### Removed
+## 10.0.6 
 
+
+### Fixed
+- Allow agent mode in biometric kyc
 ## 10.0.5
```
Comment (mldangelo) [inline CHANGELOG.md:15]:
```suggestion
- Allow agent mode in biometric kyc

## 10.0.5
```

Comment (mldangelo) [inline CHANGELOG.md:15]:
```suggestion
- Allow agent mode in biometric kyc

## 10.0.5
```

**Review by jumaallan — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## 10.0.6 (unreleased)
+## 10.0.7 (unreleased)
```
Comment (mldangelo) [inline CHANGELOG.md:1]: We shoudn't anticipate the next release here until you are ready to make it. See https://keepachangelog.com/en/1.1.0/ 


```suggestion
## [Unreleased]
```

**Review by mldangelo — APPROVED** [review]
Two nits. LTGM! Thanks

**Pull request merged**